TYPO3.Flow/Classes/TYPO3/Flow/Object/DependencyInjection/ProxyClassBuilder.php

```diff
@@ -243,35 +243,43 @@ protected function buildConstructorInjectionCode(Configuration $objectConfigurat
     {
         $assignments = array();
 
-        $argumentConfigurations = $objectConfiguration->getArguments();
+        $configurationArguments = $objectConfiguration->getArguments();
```

Comment:
I think the old name is better. It's about the "argument  configuration", not about "configuration arguments".